include/rfb.js

```diff
@@ -432,7 +432,21 @@ var RFB;
          *   ClientInitialization - not triggered by server message
          *   ServerInitialization (to normal)
          */
+        _stateQueue: new Array(),
         _updateState: function (state, statusMsg) {
+            /* Recursion protection */
```

Comment:
in which states can we recurse like this?  It would be nice to have a comment describing this.